hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -593,7 +593,20 @@
     },
     { name: "otbn",
       type: "otbn",
-      clock_srcs: {clk_i: "main", clk_edn_i: "main", clk_otp_i: "io_div4"},
+      clock_srcs: {
```

Comment:
@rswarbrick 
@GregAC 
this is a recent-ish addition to clocks. But now we can bind each clock separately to a group.
So with this setup, we no longer need to have an `idle` for otp clocks, which is really a "secure clock" that never turns off. 

In this version, the `idle_otp_o` just isn't hooked-up, but we could also just remove it completely.